codespell_lib/tests/test_dictionary.py

```diff
@@ -12,7 +12,8 @@
 
 try:
     import aspell
-    speller = aspell.Speller('lang', 'en')
+    #speller = aspell.Speller('lang', 'en')
+    speller = aspell.Speller(('lang', 'en-custom'), ('data-dir', '/home/travis/build/codespell-project/codespell/aspell6-en-custom/'))
```

Comment:
Probably makes sense to tweak this to not rely on a hardcoded `/home/travis/...` path?